app/repository/repository-detail-controller.js

```diff
@@ -27,13 +34,13 @@ angular.module('repository-detail-controller', ['registry-services', 'app-mode-s
     $scope.getNextHref = function (){
       if($scope.maxTagsPage > $scope.tagsCurrentPage){
         var nextPageNumber = $scope.tagsCurrentPage + 1;
-        return '/repository/'+$scope.repository+'/'+ $scope.tagsPerPage +'/' +nextPageNumber;
+        return '/repository/'+$scope.repository+'?tagsPerPage='+ $scope.tagsPerPage +'&tagPage=' +nextPageNumber;
```

Comment:
I'm not sure that `tagsPerPage` can be a parameter. In line 63 of `app/app.js` this route is defined as 

```javascript
when('/repository/:repositoryUser/:repositoryName/:tagsPerPage?/:tagPage?', {
```

As you can see, here it is part of the path and not a parameter.

Correct me, if I'm mistaken.